code/modules/mob/living/carbon/brain/MMI.dm

```diff
@@ -2,7 +2,7 @@
 
 /obj/item/device/mmi
 	name = "Man-Machine Interface"
-	desc = "The Warrior's bland acronym, MMI, obscures the true horror of this monstrosity."
+	desc = "The bland acronym, MMI, obscures the true horror of this monstrosity, that nevertheless has become standard-issue on Nanotrasen stations."

```

Comment:
i dont even know what "The Warrior" refers to, but its not referred to anywhere else in the code. doesn't really make sense for it to be here.